### PR TITLE
Add CRI-O OWNERS

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Syncing the CRI-O owners as opt-in.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,4 +1,7 @@
...
-  - haircommander
-  - rphillips
-  - saschagrunert
+filters:
+  .*:
+    reviewers:
+    - cri-o-reviewers
+    approvers:
+    - cri-o-approvers
+    - rphillips
```
Comment (kwilczynski) [inline OWNERS:7]: Why not put Ryan in the approvers group? :smile:

Comment (saschagrunert) [reply OWNERS:7]: @rphillips is not part of that group in CRI-O :shrug:

Comment (kwilczynski) [reply OWNERS:7]: A special snowflake, I see. :snowflake:

**Comment by saschagrunert** [comment]
@haircommander @rphillips PTAL

**Comment by haircommander** [comment]
I suspect there may be members of the crio org that aren't interested in maintaining conmon-rs. What would you think about doing an opt-in process rather than immediately including everyone?

**Comment by saschagrunert** [comment]
> I suspect there may be members of the crio org that aren't interested in maintaining conmon-rs. What would you think about doing an opt-in process rather than immediately including everyone?

Sounds good, let's start by adding @kwilczynski.

@kwilczynski are you happy to act as reviewer for this project as well?

**Comment by kwilczynski** [comment]
> > I suspect there may be members of the crio org that aren't interested in maintaining conmon-rs. What would you think about doing an opt-in process rather than immediately including everyone?
> 
> Sounds good, let's start by adding @kwilczynski.
> 
> @kwilczynski are you happy to act as reviewer for this project as well?

@saschagrunert, sounds good to me! :+1: Thank you!

**Comment by haircommander** [comment]
welcome :tada: 
/lgtm

**Pull request merged**